site/docs/4.8.2/overview/releaseNotes.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Apache BookKeeper 4.8.2 Release Notes
+---
+
+Apache BookKeeper {{ site.latest_version }} is still under developement.
```

Comment:
I don't understand this phrase

Reply:
Ah, I need to merge the #1976 first, then rebase this one.